modules/cluster-logging-collector-log-forward-syslog.adoc

```diff
@@ -80,8 +79,9 @@ spec:
 <9> Specify which log types should be forwarded using that pipeline: `application,` `infrastructure`, or `audit`.
 <10> Specify the output to use  with that pipeline for forwarding the logs.
 <11> Optional: Specify the `default` output to forward logs to the internal Elasticsearch instance.
-<12> Optional: One or more labels to add to the logs.
-<13> Optional: Configure multiple outputs to forward logs to other external log aggregators of any supported type:
+<12> Optional: Forward structured JSON log entries as JSON objects in the `structured` field. The log entry must contain valid structured JSON; otherwise, OpenShift Logging removes the `structured` field and instead sends the log entry to the default index, `app-00000x`.
+<13> Optional: One or more labels to add to the logs.
+<14> Optional: Configure multiple outputs to forward logs to other external log aggregtors of any supported type:
 ** Optional. A name to describe the pipeline.
 ** The `inputRefs` is the log type to forward using that pipeline: `application,` `infrastructure`, or `audit`.
 ** The `outputRefs` is the name of the output to use.
```

Comment:
Sorry, I know these parts aren't part of the PR... 

Line 117:
** `6` or `lpr` for line printer subsystem --> the line printer subsystem? 
Line 135:
rfc: The RFC to be used for sending log using syslog. --> the log? Other places in the text say "send logs" or "send logs data".
Line 145:
tag: Tag specifies a record field to use as tag on the syslog message. --> as a tag?
Line 155:
A a change in the value --> double A

Reply:
Updated. Thank you. Nice catch.